pkg/loader/load.go

```diff
@@ -222,6 +232,7 @@ func (s *loaderImpl) Successes() <-chan *Txn {
 // Run will quit when all data is drained
 func (s *loaderImpl) Close() {
 	close(s.input)
+	s.cancel()
```

Comment:
in `Run()` it will still execute all the accumulated dmls (but with the context that has been canceled?)

Reply:
already remove ctx when execute sqls